documentation/modules/overview/con-overview-components-integration-rhel.adoc

```diff
@@ -0,0 +1,21 @@
+// Metadata created by nebel
+//
+
+
+[id="overview-components-integration-rhel_{context}"]
+= Integration points
```

Comment:
what is this file? I can see also `rhel` in the name but it seems more a description about module(s).

Reply:
Good point Paolo. This is a placeholder file. Now gone.